### PR TITLE
full map file parsing available

### PR DESCRIPTION
**Pull request by wraithan**

### CODE REVIEW

**Comment by wraithan** [comment]
going to bed, got a bit further on this. Still will finish it in the morning but posted up my progress.

File: arena.go [hunk 1]
```diff
@@ -76,29 +91,20 @@ func main() {
...
+    send_map(state, bots)
 
-    pick_regions(state, bots, []int64{3, 4, 7, 15, 17})
+    pick_regions(state, bots)
 
-    for state.round = 1; state.round <= 45+1; state.round++ { // TODO
+    for state.round = 1; state.round <= state.max_rounds+1; state.round++ {
         if game_over(state) {
             break
         } else if state.round == 45+1 {
```
Comment (curious-attempt-bunny) [inline arena.go]: Missed a spot.

File: arena.go [hunk 2]
```diff
@@ -176,7 +182,14 @@ func receive(bot *Bot) string {
     return line
 }
 
-func pick_regions(state *State, bots []*Bot, regions []int64) {
+func send_map(state *State, bots []*Bot) {
+
+}
+
+func pick_regions(state *State, bots []*Bot) {
+    regions := make([]int64, len(state.starting_regions), cap(state.starting_regions))
```
Comment (curious-attempt-bunny) [inline arena.go]: Not sure what `cap` does. I've been using `len`.

**Comment by curious-attempt-bunny** [comment]
It's looking good so far.

File: arena.go (hunk 2 above)
Comment (wraithan) [inline arena.go]: Does a capacity check. Could also use length. I think there might not be a difference for arrays.

**Pull request merged**